microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls.md

```diff
@@ -1,162 +1,49 @@
+1. In Microsoft Edge, navigate to `edge://apps`.
 
-To test the feature, send yourself a URL in a chat message app, or a desktop email client like Windows Mail. You can also use the Windows Run app:
+1. Find the PWA you want to disable automatic link handling for and click **Details**.
 
-1. Press `Windows logo key`+`R`.
-1. Enter a URL that your app handles.
-1. Press `Enter`.
+1. On the PWA details page, under the **Link handling** section, click the toggle button.
 
-> [!NOTE]
-> At the moment, only PWAs that were installed from the default system browser can handle URLs.
+![The edge://apps details page for the PWAmp music player app, showing where the link handling toggle button is](../media/link-handling-opt-out.png)
 
 
 <!-- ====================================================================== -->
-## Demo
-
-[DevTools Tips](https://devtoolstips.org/) is a demo Progressive Web App (PWA) that handles URLs for its own domain, so that the app opens instead of the website when a URL for its domain is used.
-
-To test URL handling on the DevTools Tips demo:
-
-1. [Enable URL handling](#enable-url-handling) in Microsoft Edge.
-
-1. Go to the [DevTools Tips](https://devtoolstips.org/) PWA.
-
-1. Click the **App available. Install DevTools Tips** (![The 'App available. Install DevTools Tips' icon..](../media/app-available-icon.png)) icon in the Address bar to install the DevTools Tips app locally.
-
-1. Press `Windows logo key` + `R` to open the Windows **Run** dialog.
-
-1. Enter a URL for one of the tips on the site, such as https://devtoolstips.org/tips/en/find-css-changes/.
-
-1. Press `Enter`.
+## Handle links from other origins with scope extensions
 
-Windows knows that your app can handle URLs, but since there may be other apps that can handle URLs, Windows asks you which app to use. Select the **DevTools Tips** app. If you want to avoid seeing this dialog every time, select **Remember my choice**.
+The manifest of a PWA defines which part of the hosting domain the PWA is scoped to. For example, the `www.contoso.com` domain name may have a PWA defined under `www.contoso.com/app` with its scope set to `/app`. In this case, all the web pages available within the `www.contoso.com/app` path are part of the PWA scope. However the web pages within the `www.contoso.com/foo` path are not part of the PWA scope. Furthermore, web pages available at `bar.contoso.com/app` or `www.contoso.co.uk` are also not part of the PWA scope.
 
-> [!div class="mx-imgBorder"]
-> ![Selecting an application to handle URLs on Windows..](../media/devtools-tips-url-handling-app-selection.png)
+Scope extensions make it possible for a PWA to capture navigation to paths, subdomains, or even sites other than its own scope. This can be useful for PWAs that span multiple domains for localization purposes for examples. E.g. a PWA may span `contoso.com`, `contoso.co.uk`, and `contoso.fr`.
```

Comment:
```suggestion
Scope extensions make it possible for a PWA to capture navigation to paths, subdomains, or even sites other than its own scope. This can be useful for PWAs that span multiple domains for localization purposes.  For example, a PWA may span `contoso.com`, `contoso.co.uk`, and `contoso.fr`.
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg